src/components/alert-box/AlertBox.tsx

```diff
@@ -0,0 +1,62 @@
+import { useTheme } from 'native-base';
+import React from 'react';
+import { Modal, View, Text, TouchableOpacity } from 'react-native';
+
+import { AlertStyles } from './alert.styles';
+import { AlertType } from './alert.types';
+
+interface Props {
+  confirmText: string;
+  message: string;
+  onClose: () => void;
+  type: AlertType;
+  title: string;
+}
+
+const SYMBOL = {
+  [AlertType.DANGER]: '╳',
+  [AlertType.INFO]: '💡',
+  [AlertType.SUCCESS]: '✔',
+  [AlertType.WARNING]: '⚠',
+};
+
+export const AlertBox: React.FC<Props> = ({ type, title, message, onClose, confirmText }) => {
+  const { colors } = useTheme();
+  const styles = AlertStyles();
+  const getAlertColor = () => {
+    switch (type) {
+      case AlertType.DANGER:
+        return colors.error[500];
+      case AlertType.SUCCESS:
+        return colors.success[500];
+      case AlertType.INFO:
+        return colors.info[500];
+      case AlertType.WARNING:
+        return colors.warning[500];
+      default:
+        return colors.gray[400];
+    }
+  };
+  const color = getAlertColor();
+
+  return (
+    <Modal transparent animationType="fade">
+      <View style={styles.overlay}>
```

Comment:
It is hard to understand the role of each nested `View` and makes it hard for future developer to reason with this markup.

Kindly check [How to build maintainable frontends that future dev's would thank you for](https://www.loom.com/share/c4352f7c0be949e8bc0377dbcba15daa) on how to create semantic markup that is easy to reason with